.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,122 @@
+name: Bug Report
```

```suggestion
name: Request issue
```

This is a form for issues with directions/matrix/isochrone requests.
As such, the name _Bug report_ is a bit misleading.

---

I don't see why "Bug Report" wouldn't work in that case. Imo "Bug Report" is quite an obvious and informal term, even for users that are not so used to our terms or very technical. "Request issue" sounds so formal ^^.

---

The structure of the template is targeted towards issues arising from GET/POST requests.

"Bug report" is general enough that I'd use it to describe any bug - in particular bugs or problems with my (docker) setup. If that's my issue, all the required fields don't make any sense at all.
But I am still reporting a bug, so that would confuse me.

Maybe "API request issue" would be more fitting?

---

True. What do you think about just `API Issue`?

`Other issue (Simple)` is anyway still available as a fallback for other issues.

---

Done in 7195fdd88cdfd744d4e4d945d5133580210b7d8c